test/unit_test/api/db/services/test_task_progress_update_rule.py

```
@@ -0,0 +1,274 @@
+    n_positional_no_default = len(positional) - n_positional_defaults
+    defaults = {}
+    for i, arg in enumerate(positional):
+        if i < n_positional_no_default:
+            defaults[arg.arg] = inspect.Parameter.empty
+        else:
+            default_node = args.defaults[i - n_positional_no_default]
+            try:
+                defaults[arg.arg] = ast.literal_eval(default_node)
+            except (ValueError, SyntaxError):
+                defaults[arg.arg] = ast.unparse(default_node)
+    for arg, default_node in zip(args.kwonlyargs, args.kw_defaults or ()):
```

Ruff (0.16.1)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)